tests/graph.py

```diff
@@ -0,0 +1,37 @@
+from supporting_components.graph_io import *
+from algorithms.color_refinement import color_refinement
+
+
+def test_copy(graph):
+    """
+    Tests the copy method of the graph
+    :param graph: The graph to copy
+    :return: Whether or not the graph is copied correctly
+    """
+    copy = graph.copy()
+    return copy.is_equal(graph)
+
+
+files = ['colorref_smallexample_4_7', 'colorref_smallexample_6_15', 'colorref_smallexample_2_49']
```

Comment:
Is het nodig om al deze grafen te testen? Ik denk dat van 1 file ook wel genoeg is.

Reply:
Is een kleine moeite, ga wel de prints iets korter maken.